catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchSource.java

```diff
@@ -179,6 +180,24 @@ public OpenSearchSource(
     this.encryptionService = encryptionService;
     this.openSearchParser = openSearchParser;
     this.openSearchFilterVisitor = openSearchFilterVisitor;
+    this.foreignMarkupBiConsumer = (elements, sourceResponse) -> {};
```

Comment:
Can we do some constructor chaining here for better maintainability

Reply:
also... seems like a hard read for an empty map

Reply:
moved to ctor chaining